src/modinfowithconflictinfo.cpp

```diff
@@ -99,9 +99,15 @@ ModInfoWithConflictInfo::Conflicts ModInfoWithConflictInfo::doConflictCheck() co
   bool providesAnything = false;
   bool hasHiddenFiles   = false;
 
-  int dataID = 0;
+  QList<int> dataIDs;
```

Comment:
Maybe a set would be more efficient (`std::set` or `std::unordered_set`)?

Reply:
`QList` stores things contiguously in Qt 6, so its improved cache hit rate should beat the theoretical asymptotic performance improvement of `std::set` and `std::unordered_set` at least up to a hundred or so entries.

Reply:
Actually I just tested and `std::unordered_set` is much worse, so indeed a simple `std::vector` would be the best, my bad.

Reply:
I'll make the change tonight.